WordPress/WordPressStatsWidgets/LockScreenWidgets/Views/LockScreenSingleStatView.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+import WidgetKit
+
+@available(iOS 16.0, *)
+struct LockScreenSingleStatView: View {
+    @Environment(\.widgetFamily) var family: WidgetFamily
+    let viewModel: LockScreenSingleStatViewModel
+
+    var body: some View {
+        if family == .accessoryRectangular {
+            ZStack {
+                AccessoryWidgetBackground().cornerRadius(8)
+                VStack(alignment: .leading) {
+                    Text(viewModel.siteName)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .font(.system(size: 11))
+                        .lineLimit(1)
+                    Text(viewModel.value)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .font(.system(size: 20, weight: .bold))
+                        .foregroundColor(.white)
```

Comment:
Maybe we could introduce `minimumScaleFactor` here? Although a number is unlikely to be so large it's cheap to avoid such corner-cases.

Reply:
Nice suggestion, sorry for I forgetting it, this point was also raised by the designer as well.
I temporarily choose 0.8 for 11 font size and 0.5 for 20 font size, I think if too smaller might become illegible.
feel free to give a better number of suggestions.